zookeeper-client/zookeeper-client-c/src/cli.c

```diff
@@ -834,6 +834,8 @@ int main(int argc, char **argv) {
                 "                                 command 'verbose' later to activate debug logs in the cli shell)\n\n",
                 argv[0]);
         fprintf(stderr,
+                "Version: ZooKeeper cli (c client) version %s\n",
+                ZOO_VERSION);
                 "Version: ZooKeeper cli (c client) version %d.%d.%d\n",
```

Comment:
Why not remove the old print for the version?

Reply:
I have used the github wizard to resolve a conflict in this position and I failed :-( 

I will update the patch ASAP, sorry

Reply:
done, thanks

Reply:
No problem, I just wasn't sure if there's any reason to keep or it was by accident. Thanks!